<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEditItemScreen</class>
 <widget class="QMainWindow" name="NewEditItemScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>473</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="Austria"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="Austria"/>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="Austria"/>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_item">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>{New/Edit} Item</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_articlenumber">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>{component}: {articleNumber}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_articleNumber">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Article No.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_errorArticleNumber">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Already in Database!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_itemTyoe">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_itemType">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_itemVersion">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_itemVersion">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Version:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txb_articleNumber">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txb_propertyValue">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_refernece">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_unit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="dtv_itemData">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>451</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_okay">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_articleNumber_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Item Data:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
